Tests/DangerTests/GitTests.swift

```diff
@@ -1,6 +1,130 @@
-import Danger
+@testable import Danger
+import DangerFixtures
 import XCTest
 
 final class GitTests: XCTestCase {
-    func test() {}
+    private var git: Git { githubFixtureDSL.git }
+
+    func testDecodesCommits() {
```

Comment:
<!--
  1 failure:  Function body sho...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/GitTests.swift;
  Line: 8;
-->
- :no_entry_sign: Function body should span 40 lines or less excluding comments and whitespace: currently spans 104 lines (`function_body_length`)